VSCode refuses to start

I get on current master:

    vscode-dotty/test/extension.test.ts(15,1): error TS2304: Cannot find name 'suite'.

    15 suite("vscode-dotty tests", function () {
       ~~~~~


    vscode-dotty/test/extension.test.ts(18,5): error TS2304: Cannot find name 'test'.

    18     test("dummy test", function() {
           ~~~~

    [error] Running command "/Users/odersky/workspace/dotty/vscode-dotty/node_modules/.bin/tsc --pretty --project /Users/odersky/workspace/dotty/vscode-dotty" failed.
    [error] (vscode-dotty/compile:compile) Running command "/Users/odersky/workspace/dotty/vscode-dotty/node_modules/.bin/tsc --pretty --project 
    /Users/odersky/workspace/dotty/vscode-dotty" failed.
    [error] Total time: 1 s, completed Mar 22, 2018 7:16:38 PM

@smarter Any idea what could be the cause of this?

--- a/vscode-dotty/package.json
+++ b/vscode-dotty/package.json
@@ -62,6 +62,7 @@
   },
   "devDependencies": {
     "@types/node": "^8.5.8",
+    "@types/mocha": "^5.0.0",
     "typescript": "^2.6.2",
     "vscode": "^1.1.10"
   }

